packages/raccoon-city-ui/src/components/Main/ChessGrid/ChessFloorView/FloorViewsParts/FlatInfoBar.tsx

```diff
@@ -12,7 +12,7 @@ export const FlatInfoBar = ({info}: Props) => (
             <>
                 <span>{`№${info.flatNumber}`}</span>
                 <span>{`Статус: ${FLAT_STATUSES.find((statuses) => statuses.value === info.status)?.label}`}</span>
-                <span>{`Цена: ${info.price}`}</span>
+                {info.status !== 'SOLD_OUT' && <span>{`Цена: ${info.price}`}</span>}
```

Comment:
nit

We should be able in the future to handle the case when the flat was sold out with sale price